mtags-interfaces/src/main/java/scala/meta/pc/CompletionItemPriority.java

```diff
@@ -0,0 +1,6 @@
+package scala.meta.pc;
+
+public interface CompletionItemPriority {
+
+    Integer workspaceMemberPriority(String buildTargetIdentifier, String symbol);
```

Comment:
Let's add docs what is expected by returning an integer here. What happens when the number is lower and higher.